[DM-4568]: Created CL for removal of deprecated function from context-dashboard service (#3601)
SoftwareAG/c8y-docs: content/change-logs/device-management/ui-c8y-1022-0-0-removed-deprecated-getnameddashboardorcreate-from-context-dashboard-service.md

---
date: ""
title: Removed deprecated getNamedDashboardOrCreate from context-dashboard service
product_area: Device management & connectivity
change_type:
  - value: change-2c7RdTdXo4
    label: Improvement
component:
  - value: component--KIsStyzM

>     label: Device Management app

Reviewer: This should probably be "REST API" in the platform services product area? And is this a breaking change?

Author: Yes, it is actual a WebSDK component with a API change type. I have changed it accordingly, and yes it is a removal of a deprecated function, so a breaking change yes.

Reviewer: Thanks, Dafina. Can you please move the file into the App enablement folder?

build_artifact:
  - value: tc-pjJiURv9Y
    label: ui-c8y
ticket: DM-4568
version: 1022.0.0
---

> We’ve cleaned up the Dashboard API by removing the old, deprecated `getNamedDashboardOrCreate` method from the `context-dashboard` service. Going forward, use the newer `getDashboard()` call.

Reviewer: ```suggestion
The deprecated `getNamedDashboardOrCreate` method has been removed from the `context-dashboard` service in the Dashboard API. In the future, the `getDashboard()` call must be used instead.
```
